src/main/resources/theme/vault/map/maps_management.fxml

```diff
@@ -0,0 +1,97 @@
+                <ColumnConstraints halignment="RIGHT" hgrow="ALWAYS"/>
+            </columnConstraints>
+            <rowConstraints>
+                <RowConstraints minHeight="10.0"/>
+            </rowConstraints>
+            <children>
+                <ChoiceBox fx:id="filterMapsChoiceBox" maxWidth="170.0" minWidth="170.0">
+                    <GridPane.margin>
+                        <Insets/>
+                    </GridPane.margin>
+                </ChoiceBox>
+                <HBox alignment="CENTER_RIGHT" nodeOrientation="LEFT_TO_RIGHT" prefHeight="100.0" prefWidth="200.0"
+                      GridPane.columnIndex="1">
+                    <children>
+                        <Button fx:id="removeButton" disable="true" mnemonicParsing="false"
+                                onAction="#onRemoveButtonClicked" style="-fx-background-color: darkred;">
+                            <graphic>
+                                <Region>
+                                    <styleClass>
+                                        <String fx:value="icon"/>
+                                        <String fx:value="uninstall-icon"/>
+                                    </styleClass>
+                                </Region>
+                            </graphic>
+                        </Button>
+                        <Label text="%management.maps.remove">
+                            <HBox.margin>
+                                <Insets left="10.0"/>
+                            </HBox.margin>
+                        </Label>
+                    </children>
+                    <GridPane.margin>
+                        <Insets right="5.0"/>
+                    </GridPane.margin>
+                </HBox>
+            </children>
+        </GridPane>
+        <Button fx:id="closeButton" mnemonicParsing="false" onAction="#onCloseButtonClicked"
+                text="%management.maps.closeButton.label" GridPane.halignment="RIGHT" GridPane.rowIndex="3"
+                GridPane.valignment="CENTER">
+            <GridPane.margin>
+                <Insets top="30.0"/>
+            </GridPane.margin>
+        </Button>
+        <TableView fx:id="tableView" style="-fx-border-color: #2F2F2F   ; -fx-border-width: 5;" GridPane.rowIndex="1">
+            <placeholder>
+                <Label/>
+            </placeholder>
+            <columns>
+                <TableColumn fx:id="checkboxColumn" editable="false" maxWidth="50.0" minWidth="50.0" prefWidth="50.0"
+                             resizable="false" sortable="false"/>
+                <TableColumn fx:id="previewMapColumn" editable="false" maxWidth="60.0" minWidth="60.0" prefWidth="60.0"
+                             resizable="false" sortable="false" text="%management.maps.map"/>
+                <TableColumn fx:id="mapNameColumn" editable="false" minWidth="233.0" prefWidth="339.0" sortable="false"
+                             text="%management.maps.name"/>
```

Comment:
Why make it not resizable? This could make it hard for some users with different screen sizes.